app/assets/stylesheets/modules/accessibility.scss

```diff
@@ -5,3 +5,8 @@
     outline: $input-focus-border-color auto 5px;
   }
 }
+
+/* Needed for focus ring to wrap around the element */
+.browse-landing a {
```

Comment:
We also need this applied to `.browse-categories a` for widgets that embed browse categories, for example: https://exhibits.stanford.edu/richard-weiland/feature/ric-s-story